lib/Common/Memory/RecyclerPointers.h

```diff
@@ -489,21 +489,26 @@ template <class Policy>
 struct _QuickSortImpl
 {
     template<class T, class Comparer>
-    static void Sort(T* arr, size_t count, const Comparer& comparer, void* context)
+    static void Sort(T* arr, size_t count, const Comparer& comparer, void* context, size_t elementSize = sizeof(T))
     {
+#ifndef _WIN32
+        JsUtil::QuickSort<Policy, char, Comparer>::Sort((char*)arr, count, elementSize, comparer, context);
```

Comment:
>JsUtil::QuickSort<Policy, char, Comparer>::Sort((char*)arr, count, elementSize, comparer, context); [](start = 8, length = 99)

Related to other comment. This has type and should be able to call like "JsUtil::QuickSort<Policy, T, Comparer>::Sort(arr, count, comparer, context)", below WB instantiation could call like "JsUtil::QuickSort<_write_barrier_policy, T, Comparer>::Sort(arr, count, comparer, context)", without casting and tweaking elementSize.